python/lsst/ts/logging_and_reporting/web_app/services/scheduler_service.py

```diff
@@ -1030,3 +1034,62 @@ def my_create_metric_visit_map_grid(
         return map_grid
 
     return None
+
+
+def get_expected_exposures(
+    dayObsStart: int,
+    dayObsEnd: int,
+) -> dict:
+    """
+    Retrieve the expected exposures for Simonyi for a specified range
+    of observation nights.
```

Comment:
```suggestion
    """Retrieve the expected exposures for Simonyi for a specified range
    of observation nights.
```
https://developer.lsst.io/python/numpydoc.html#docstrings-should-begin-with-and-terminate-with-on-its-own-line.

Please check this one and other cases 🙏